packages/terra-form-input/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed
+* Fixed Placeholder color for IE by adding vendor specific style rule
```

Comment:
Maybe consider updating this to something that mentions the issue - 

`Faded the placeholder text on IE11 by adding vendor specific style rule.`

Reply:
ec4b0bd518032515927780abeb5ae72600633807